samples/msal-angular-v2-samples/angular10-sample-app/test/home.spec.ts

```diff
@@ -47,6 +47,7 @@ describe('/ (Home Page)', () => {
     beforeEach(async () => {
         context = await browser.createIncognitoBrowserContext();
         page = await context.newPage();
+        page.setDefaultTimeout(5000);
```

Comment:
Does it throw if we timeout?

Reply:
This only sets the timeout value. But any navigation operations done on the page object will throw if they don't complete in 5 seconds. In some places this error is caught, but this is to prevent "stuck" navigations that will hold up the test for 30 seconds and cause the test to fail due to test timeout.

Reply:
Got it. Thanks.